examples/sveltekit/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "useTabs": true,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "printWidth": 100
+}
```

Comment:
@fbjork @jvidalv 
Let's move prettier config to the root of repo, so all examples and templates share the same styling?

Reply:
Sure

Reply:
Can we do it on another pr?

Reply:
Sure, I've opened PR: https://github.com/grafbase/grafbase/pull/43

Reply:
Nice, thanks!